cli/azd/pkg/project/service_target.go

```diff
@@ -25,6 +25,19 @@ const (
 	AksTarget           ServiceTargetKind = "aks"
 )
 
+func parseServiceHost(kind ServiceTargetKind) (ServiceTargetKind, error) {
+	switch kind {
+	case AppServiceTarget,
+		ContainerAppTarget,
+		AzureFunctionTarget,
+		StaticWebAppTarget,
+		AksTarget:
+		return kind, nil
+	}
```

Comment:
Similar comment to above about creating a exported package var with an array/map of supported service targets and perform an array contains or similar.

Reply:
I can see why that could be useful, but I'm going to hold off until we have something that we want to expose in-mind.